.env.production

```diff
@@ -1,6 +1,6 @@
 REACT_APP_FATHOM_SITE_ID=KKJYZYXY
 REACT_APP_FATHOM_SITE_SCRIPT_URL=https://gold-stunning.swapr.site/script.js
-REACT_APP_STACKLY_DESIGN_VERSION=old
+REACT_APP_STACKLY_DESIGN_VERSION=new
```

Comment:
I think we should remove this variable now and just show the new banner

Reply:
Yes! This change is faster for release purposes, talked about this with @0xVenky and he agreed on me creating a follow-up task to drop all the unused code in another task (post-release):

https://linear.app/swaprhq/issue/SWA-101/clean-all-unused-code-and-env-vars-for-stackly-toast

Reply:
ok!